src/main.py

```diff
@@ -103,14 +101,14 @@ async def on_list(
         list_channel = interaction.channel
 
     async with list_task_control_lock:
-        bc = await create_controller(client, interaction, list_channel)
-        controllers[interaction.guild_id] = bc
+        bc = await bust.create_controller(client, interaction, list_channel)
+        bust.controllers[interaction.guild_id] = bc
 
 
 # Bust command
-@client.slash_command(dm_permission=False)
+@client.slash_command(name="bust", dm_permission=False)
 @application_checks.has_role(config.dj_role_name)
-async def bust(
+async def do_bust(
```

Comment:
seems a little odd calling it `do_bust`, maybe call it something like `bust_action` or `launch_bust`

Reply:
probably on_bust since we have on_list, will change later